app/assets/javascripts/activeadmin_addons/date-time-picker.js

```diff
@@ -16,7 +16,7 @@ $(function() {
       defaultSelect: false,
       validateOnBlur: false
     };
-    entries = $(container).find('input.date-time-picker');
+    entries = $(container).find('input.date-time-picker, textarea.date-time-picker');
```

Comment:
@ldlsegovia this error is because `$` is not defined as a global var. Should I add it or find  I have to find another way to do it?